lms/djangoapps/support/static/support/js/spec/certificates_spec.js

```diff
@@ -0,0 +1,163 @@
+define([
+    'jquery',
+    'common/js/spec_helpers/ajax_helpers',
+    'support/js/views/certificates'
+], function($, AjaxHelpers, CertificatesView) {
+    'use strict';
+
+    describe('CertificatesView', function() {
+
+        var view = null;
+
+        var SEARCH_RESULTS = [
+            {
+                "username": "student",
+                "status": "notpassing",
+                "created": "2015-08-05T17:32:25+00:00",
+                "grade": "0.0",
+                "type": "honor",
+                "course_key": "course-v1:edX+DemoX+Demo_Course",
+                "download_url": null,
+                "modified": "2015-08-06T19:47:07+00:00"
+            },
+            {
+                "username": "student",
+                "status": "downloadable",
+                "created": "2015-08-05T17:53:33+00:00",
+                "grade": "1.0",
+                "type": "verified",
+                "course_key": "edx/test/2015",
+                "download_url": "http://www.example.com/certificate.pdf",
+                "modified": "2015-08-06T19:47:05+00:00"
+            },
+        ];
+
+        var getSearchResults = function() {
+            var results = [];
+
+            $(".certificates-results tr").each(function(rowIndex, rowValue) {
+                var columns = [];
+                $(rowValue).children("td").each(function(colIndex, colValue) {
+                    columns[colIndex] = $(colValue).html();
+                });
+
+                if (columns.length > 0) {
+                    results.push(columns);
+                }
+            });
+
+            return results;
+        };
+
+        var searchFor = function(query, requests, response) {
+            // Enter the search term and submit
+            view.setUserQuery(query);
+            view.triggerSearch();
+
+            // Simulate a response from the server
+            AjaxHelpers.expectJsonRequest(requests, "GET", "/certificates/search?query=student@example.com");
+            AjaxHelpers.respondWithJson(requests, response);
+        };
+
+        var regenerateCerts = function(username, courseKey) {
+            var sel = '.btn-cert-regenerate[data-course-key="' + courseKey + '"]';
+            $(sel).click();
+        };
```

Comment:
@AlasdairSwan I'm not quite sure how this should look.  Is there an issue with putting these `var` declarations at the top of the anonymous function passed as an arg to `describe()`?